pyfar/signals/deterministic.py

```diff
@@ -35,15 +25,16 @@ def sine(frequency, n_samples, amplitude=1, phase=0, sampling_rate=44100,
 
     Returns
     -------
-    signal : pyfar.Signal
+    signal : Signal
         The sine as a Signal object. The Signal is in the time domain and has
-        the 'rms' FFT normalization (see pyfar.fft.normalization). The exact
-        frequency is written to Signal.comment.
+        the 'rms' FFT normalization (see
+        :py:func:`pyfar.dsp.fft.normalization`). The exact frequency, amplitude
+        and phase are written to signal.comment.
 
     Note
     ----
-    The parameters frequency, amplitude, and samples must all be scalars
-    or of the same shape.
+    The parameters frequency, amplitude, and phase must must be scalars
+    and/or array likes of the same shape.
```

Comment:
`Note` looks different than `Notes`, is this intended?

Reply:
Not intended and changed to Notes :)